### PR TITLE
Update get function to filter by org

### PR DESCRIPTION
**Pull request by lucaslinhares**
Update get function to filter globals by org to avoid that get method brings all globals data from another org.

### CODE REVIEW

File: weni/internal/globals/views.py [hunk 1]
```diff
@@ -15,9 +18,21 @@ class GlobalViewSet(
...
 
+    def get_queryset(self):
+        queryset = Global.objects.filter(is_active=True)
+        org = self.request.query_params.get("org")
+
+        try:
+            org_object = Org.objects.get(uuid=org)
+            queryset = queryset.filter(org=org_object)
+
+        except:
```
Comment (elitonzky) [inline weni/internal/globals/views.py:31]: It's good practice to specify exceptions, you can do it like this:
`    try:
          org_object = Org.objects.get(uuid=org)
          queryset = queryset.filter(org=org_object)
          return queryset
    except Org.DoesNotExist:
        raise ValidationError(detail={"message":"Org not found"})`

File: weni/internal/globals/views.py [hunk 2]
```diff
@@ -15,9 +18,21 @@ class GlobalViewSet(
...
+            raise ValidationError(detail={"message":"Org not found"})
```
Comment (elitonzky) [inline weni/internal/globals/views.py:32]: Missing whitespace after ':'

File: weni/internal/globals/views.py [hunk 3]
```diff
@@ -1,8 +1,11 @@
 from rest_framework import mixins
 from rest_framework import status
 from rest_framework.response import Response
+from django.shortcuts import get_object_or_404
```
[the remark this answers is not in the document]
Comment (lucaslinhares) [reply weni/internal/globals/views.py]: Done

File: weni/internal/globals/views.py (hunk 1 above)
Comment (lucaslinhares) [reply weni/internal/globals/views.py]: Done

File: weni/internal/globals/views.py (hunk 2 above)
Comment (lucaslinhares) [reply weni/internal/globals/views.py]: Done

**Review by elitonzky — APPROVED** [review]

**Pull request merged**